source/SIL.AppBuilder.Portal.Frontend/src/ui/styles/semantic-ui-overrides.scss

```diff
@@ -63,6 +63,17 @@ Inputs
     border-bottom: solid 1px $input-border-color;
     padding: 5px;
   }
+
+  textarea {
+    font-family: Montserrat;
```

Comment:
a lot of this is redundant, yeah? Please try to re-use, rather than re-create :)